File: app/src/main/java/com/example/pam_app/utils/DecimalPercentageFormatter.java
package com.example.pam_app.utils;

import com.github.mikephil.charting.formatter.ValueFormatter;

import java.text.DecimalFormat;

public class DecimalPercentageFormatter extends ValueFormatter {

    protected DecimalFormat mFormat;

    public DecimalPercentageFormatter(DecimalFormat format) {
        this.mFormat = format;
    }

    @Override
    public String getFormattedValue(float value) {
        if(value < 3) return "";
        return mFormat.format(value) + " %";
    }
}

File: app/src/main/java/com/example/pam_app/utils/DecimalPercentageFormatter.kt
package com.example.pam_app.utils

import com.github.mikephil.charting.formatter.ValueFormatter
import java.text.DecimalFormat
import kotlin.jvm.Synchronized

class DecimalPercentageFormatter(protected var mFormat: DecimalFormat) : ValueFormatter() {
    override fun getFormattedValue(value: Float): String {
        return if (value < 3) "" else mFormat.format(value.toDouble()) + " %"
    }

}